src/documentation/0052-nodejs-streams/index.md

```diff
@@ -249,13 +249,13 @@ then implement `_transform`:
 
 ```js
 TransformStream._transform = (chunk, encoding, callback) => {
-  console.log(chunk.toString().toUpperCase());
+  push(chunk.toString().toUpperCase());
```

Comment:
What is `push` function here?

Reply:
https://nodejs.org/api/stream.html#stream_readable_push_chunk_encoding

As it pertains to transform, it is documented here: https://nodejs.org/api/stream.html#stream_transform_transform_chunk_encoding_callback

Reply:
You just mean adding `this`, right? If there's more let me know. Thanks.

Reply:
Yes, thank you